bin/gen_csr_access_test.py

```diff
@@ -71,11 +74,19 @@
 if args.dry_run:
     print('{}'.format(args))
 
+
 script_path      = os.path.join(topdir, '{}/vendor_lib/google/riscv-dv/scripts/gen_csr_test.py'.format(args.core.lower()))
-yaml_file_path   = os.path.join(topdir, '{}'.format(args.core.lower()) + '/env/corev-dv/{}'.format(args.core.lower()) + '_csr_template.yaml')
 template_path    = os.path.join(topdir, './bin/templates/csr_access_test_template.S')
 output_yaml_path = ""
 
+if (args.input_yaml_path != None):
+    yaml_file_path   = '{}'.format(args.input_yaml_path.lower())
```

Comment:
Why is there `.lower()` on this?

Reply:
Original though: because the other path uses lower when handling the core argument. But this is a good point. paths don't necessarily use lower case.